_overviews/jdk-compatibility/overview.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: multipage-overview
+title: Overview
+
+discourse: true
+
+partof: jdk-compatibility
+overview-name: JDK Compatibility
+
+num: 1
+permalink: /overviews/jdk-compatibility/:title.html
+---
+
+Scala runs primarily on the Java Virtual Machine (JVM). As Scala and the JVM improve independently over time, Scala drops compatibility with older versions of the Java Developer Kit (JDK) in order to focus development efforts on supporting new JVM features that benefit Scala.
+
+This table shows the first Scala release in each series that functions on each JDK.
+
+| JDK version | First Scala release supporting JDK version per series |
+|:-----------:|:-----------------------------------------------------|
+| 9           | 2.12.4, 2.11.12                                      |
+| 8           | 2.12.0, 2.11.0, 2.10.7                               |
```

Comment:
earlier 2.10.x releases work fine on Java 8

Reply:
Addressed in https://github.com/scala/docs.scala-lang/pull/1005/commits/979f07aa1bcdf3db5faa962c54479254e8e71d3d